### PR TITLE
GRIDEDIT-1495: updated badges in readme

### PR DESCRIPTION
**Pull request by veenstrajelmer**
Tiny PR, just so we can walk trough the branch>>PR>>review>>merge process once. The preview of the updated readme is available here: https://github.com/Deltares/MeshKernelPy/tree/update-readme-badges

Checks:
- [x] branch is up to date with main
- [ ] tests are passing

### CODE REVIEW

**Review by lucacarniato — APPROVED** [review]

**Comment by veenstrajelmer** [comment]
@lucacarniato I do not have merge permissions (also not after your review), so could you merge it?

**Review by lucacarniato — APPROVED** [review]

**Pull request merged**
